vyos/provider.go

```diff
@@ -49,6 +49,15 @@ func Provider() *schema.Provider {
 			"vyos_dhcp_server_subnet":              resourceInfoDhcpServerSubnet().ResourceSchema,
 			"vyos_dhcp_server_subnet_address_pool": resourceInfoDhcpServerSubnetAddressPool().ResourceSchema,
 			"vyos_vrrp_group":                      resourceInfoVrrpGroup().ResourceSchema,
+			"vyos_interface_ethernet":              resourceInterfaceEthernet().ResourceSchema,
+			"vyos_interface_ethernet_vif":          resourceInterfaceEthernetVif().ResourceSchema,
+			"vyos_interface_pseudo_ethernet":       resourceInterfacePseudoEthernet().ResourceSchema,
+			"vyos_intercafe_pseudo_ethernet_vif":   resourceInterfacePseudoEthernetVif().ResourceSchema,
```

Comment:
Misspelling of interface in resource name

Reply:
ty